pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -220,16 +212,7 @@ func (c *kubeControllersComponent) Objects() ([]client.Object, []client.Object)
 		c.controllersDeployment(),
 	}
 	objectsToDelete := []client.Object{}
-	if c.renderManagerInternalSecret {
```

Comment:
This is just left over cruft that's not needed anymore since we've moved to using the gateway, right?